melm_bringup/launch/laser.launch.py

```diff
@@ -40,10 +40,10 @@ def generate_launch_description():
             {'enable_box_crop_func': True},
             {'angle_crop_min': 175.0},
             {'angle_crop_max': 210.0},
-            {'x_crop_min': -0.30},
-            {'x_crop_max': 0.20},
-            {'y_crop_min': -0.09},
-            {'y_crop_max': 0.09},
+            {'x_crop_min': -0.55},
+            {'x_crop_max': 0.45},
+            {'y_crop_min': -0.34},
+            {'y_crop_max': 0.34},
```

Comment:
```suggestion
            {'x_crop_min': -0.3},
            {'x_crop_max': 0.3},
            {'y_crop_min': -0.3},
            {'y_crop_max': 0.3},
```

Reply:
Please change to +/- .3m or roughly  `y_min/max = -/+ (measured_width_melm)/2` and same thing for x with measured length. I think it's "close" to .6m in each dimension?